rc/tools/clang.kak

```diff
@@ -173,6 +173,8 @@ define-command clang-diagnostics-next -docstring "Jump to the next line that con
     evaluate-commands %sh{
         eval "set -- ${kak_quoted_opt_clang_errors}"
         shift # skip timestamp
+        unset line
+        unset first_line
```

Comment:
Do we need those? `$line` is explicitly assigned to on line 186, and `$first_line` then doesn't need to be initialised.

Reply:
If `$line` was already set from the environment and is never set in the loop, then
`line=${line-$first_line}` will not change `$line`, so the inherited value will be executed.

Similarly, if `$first_line` is inherited from the environment, then its value will never change, and it could be assigned to `$line`.

Apparently `${var-default}` expands to `default` if `$var` is unset,
whereas `${var:-default}` expands to `default` if `$var` is unset or empty.

That's why I use `unset` over `line=""`. Maybe it should be rewritten.